ankihub/gui/web/reviewer_buttons.js

```diff
@@ -33,7 +34,8 @@ class AnkiHubReviewerButtons {
             },
             {
                 name: "chatbot",
-                iconPath: "/_chatbot_icon.svg",
+                iconPath: this.isPremium ? "/_chatbot_icon.svg" : "/_chatbot_icon_sleeping.svg",
+                iconPathDarkTheme: this.isPremium ? null : "/_chatbot_icon_sleeping_dark_theme.svg",
```

Comment:
:question: why the icon is `null`? is the file missing? should we ask the designers for the dark version?

Reply:
No separate icon is required because there's logic in the file to change the foreground/background colors and that works for the this icon but not for the sleeping version (thus the separate _chatbot_icon_sleeping_dark_theme.svg version)

![image](https://github.com/user-attachments/assets/f07a0d80-a981-49df-bd8d-1b358b85a9a4)